src/lib/Flasher.js

```diff
@@ -296,10 +302,12 @@ class Flasher {
         'Starting FastOTA update',
       );
     }
-
+    let chunkCount = 0;
     this._readNextChunk();
     while (this._chunk) {
       const messageToken = this._sendChunk(this._chunkIndex);
+      chunkCount += 1;
+      if (chunkCount % 50 === 0) await waitFor(500);
```

Comment:
For Photon there is generally less than 50 chunks so this code will never be run. If it does, I think 500ms is way too long of a wait time.

I think we can have small delay here of 100ms. If this is too much for the low bandwidth scenarios, the missed chunks should handle it and add a larger delay between chunks.